spec.html

```diff
@@ -27616,7 +27879,8 @@ <h1>
               1. Assert: _module_ imports a specific binding for this export.
               1. Append _e_.[[ExportName]] to _exportedNames_.
             1. For each ExportEntry Record _e_ of _module_.[[StarExportEntries]], do
-              1. Let _requestedModule_ be ? HostResolveImportedModule(_module_, _e_.[[ModuleRequest]]).
+              1. Let _requiredModule_ be GetImportedModule(_module_, _e_.[[ModuleRequest]]).
+              1. If _requiredModule_ is ~empty~, throw a *TypeError* exception.
```

Comment:
Should this be an assertion or a TypeError?

Reply:
The current spec uses ? so it's an error, but I think an assertion would be better (it's the same as point 2 of the PR description)